### DISCUSSION TITLE
How can I disable the right click paste?

### DISCUSSION THREAD

**Discussion by zawadzki6** [chrisant996/clink#704] [Q&A] [answered]
^^ I can't seem to find anything about it and would really appreciate it

**Comment by chrisant996** [comment]
Clink isn't doing that.  Your terminal program is doing that.  Refer to the documentation of the terminal program.

**Reply by zawadzki6** [reply]
I believe it is. I am using Windows Terminal and it only started happening when I installed Clink

**Reply by chrisant996** [reply, marked as answer]
> I believe it is. I am using Windows Terminal and it only started happening when I installed Clink

No, Clink doesn't care about right clicks and doesn't do anything with them.

Windows Terminal is what's responding to the right click.  Refer to Windows Terminal documentation to learn more about its features and behaviors.

Here is the setting in Windows Terminal:

![image](https://github.com/user-attachments/assets/d60d73c9-aed1-4bd2-9e06-a498b2169494)

**Comment by zawadzki6** [comment]
I found out that I am missing that option for some reason. But I managed to do it with the config file. So thanks for the help!

^^ apparently the option is experimental that's probably why it's missing
